config/webpack/webpack.config.js

```diff
@@ -108,17 +108,23 @@ const webpackConfig = (options, env, argv) => {
       }),
       new DuplicatePackageCheckerPlugin({
         showHelp: false,
+        alwaysEmitErrorsFor: [
+          'react',
+          'react-dom',
+          'react-intl',
+          'terra-base',
```

Comment:
Should terra-base be here? I'm wondering if it will break people more than we'd like (as of right now, us in terra-framework)

Reply:
![image](https://user-images.githubusercontent.com/6720991/60107336-5e6bf780-972c-11e9-83c9-101aaa9fb6d7.png)

Reply:
We could remove terra-base for the time being and only enforce usage of it through terra-application. 🤷‍♂

Reply:
Using the same logic here to remove terra-modal-manager and terra-slide-panel-manager from the list. terra-disclosure-manager is the package that really needs to be controlled, and those others have peer dependencies on it, so everything should work out.